src/gui/tray/UserLine.qml

```diff
@@ -71,27 +71,25 @@ MenuItem {
                         Layout.preferredHeight: (userLineLayout.height -16)
                         Layout.preferredWidth: (userLineLayout.height -16)
                         Rectangle {
-                            id: accountStateIndicatorBackground
-                            width: accountStateIndicator.sourceSize.width + 2
+                            id: accountStatusIndicatorBackground
+                            width: accountStatusIndicator.sourceSize.width + 2
                             height: width
                             anchors.bottom: accountAvatar.bottom
                             anchors.right: accountAvatar.right
                             color: "white"
                             radius: width*0.5
                         }
                         Image {
-                            id: accountStateIndicator
-                            source: model.isConnected
-                                    ? Style.stateOnlineImageSource
-                                    : Style.stateOfflineImageSource
+                            id: accountStatusIndicator
+                            source: model.statusIcon
                             cache: false
-                            x: accountStateIndicatorBackground.x + 1
-                            y: accountStateIndicatorBackground.y + 1
+                            x: accountStatusIndicatorBackground.x + 1
+                            y: accountStatusIndicatorBackground.y + 1
                             sourceSize.width: Style.accountAvatarStateIndicatorSize
                             sourceSize.height: Style.accountAvatarStateIndicatorSize
 
                             Accessible.role: Accessible.Indicator
-                            Accessible.name: model.isConnected ? qsTr("Account connected") : qsTr("Account not connected")
+                            Accessible.name: model.isStatusOnline ? qsTr("Current user status is online") : qsTr("Current user status is do not disturb")

```

Comment:
Ok, it might help.

Reply:
I am not sure anymore: https://doc.qt.io/qt-5/qml-qtquick-accessible.html#name-prop

Reply:
I'm not sure, what's the question? :-)

Reply:
The question is if I should add the double quotes around the words "online" and "do not disturb"  in the text of Accessible.name. The doc says "this property should be set to a simple and concise but human readable name".

Reply:
@camilasan Since this was just a suggestion, let's keep it like that. This is something that can always be changed later if needed. Let's note make things more complicated ;)